lib/Twig/Environment.php

```diff
@@ -1280,28 +1288,10 @@ protected function initExtension(Twig_ExtensionInterface $extension)
 
     protected function writeCacheFile($file, $content)
```

Comment:
this is quite hard, because we should not call it when the Null cache is used (otherwise it is not BC either)